Dockerfile

```
@@ -31,13 +31,10 @@ RUN [ -s /home/extractor/requirements.txt ] && \
     (echo 'No python modules to install' && \
      rm /home/extractor/requirements.txt)
 
-RUN (echo "installing sudo" && \
-    apt-get update && \
-    apt-get -y install sudo)
-
 RUN (echo "installing osgeo dependencies" && \
-    sudo apt-get install -y python3-gdal gdal-bin libgdal-dev gcc g++ python3.5-dev && \
-    sudo python3 -m pip install --no-cache-dir pygdal==2.2.2.*)
+    apt-get update && \
+    apt-get install -y python3-gdal gdal-bin libgdal-dev gcc g++ python3.5-dev && \
+    python3 -m pip install --no-cache-dir pygdal==2.2.2.*)
```

```suggestion
    python3 -m pip install --no-cache-dir pygdal==2.2.2.* && \
    rm /home/extractor/packages.txt && \
    apt-get autoremove -y && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*)
```
Making this suggestion since there's another change that needs to happen